### DISCUSSION TITLE
OneDrive vs MountainDuck – were are the files I have to work with?

### DISCUSSION THREAD

**Discussion by VIDEORealism** [iterate-ch/cyberduck#16638] [Q&A] [answered]
Until 2 days ago I had no problems using MoutainDuck. in the morning I was still able to work with the data in OneDrive without any problems, but when I sat down at the Mac again in the afternoon, the locally synchronised data was gone and the data on the server was no longer displayed. all that is still shown is an empty ‘Apps’ folder.

Of course I have already asked the support and sent the log, which should show that the folder, to which everything should be synchronised, is not accessible; the external drive (I don't want to fill up my system disk) is permanently connected to the mac and can be accessed without any problems.

In the meantime I have remounted the OneDrive, tried another folder for syncing, reinstalled MountainDuck - the same problem also exists on another Mac.

Any ideas beyond using an external drive as storage for the synchronised data?

Best,
Phil

**Comment by dkocher** [comment, marked as answer]
Possibly you previously worked with files in _Shared_ folder. Set the Path in the [Bookmark](https://docs.mountainduck.io/cyberduck/bookmarks/) to `/` to show the top level folders. By default, only the contents of My Files are shown [^1].

[^1]: https://docs.mountainduck.io/protocols/onedrive/#top-level-folders

**Reply by VIDEORealism** [reply]
Yes well THAT was helpful! I have no idea why the path changed in the first place ... I did not change anything in this setting, but I can sync my files again.
Thank you!
